src/app/shared/asyncapi.service.ts

```diff
@@ -50,70 +20,7 @@ export class AsyncApiService {
         }
 
         return this.http.get<ServerAsyncApi>(Endpoints.docs).pipe(map(item => {
-            this.docs = this.toAsyncApi(item);
```

Comment:
This line is important - without it each call to `getAsyncApis` (which should be renamed to `getAsyncApi`, I missed that in my previous PR) will trigger an actual http request.

Reply:
Thanks for pointing this out. Now I start to understand the pattern here